src/vsr/journal.zig

```diff
@@ -1451,11 +1450,15 @@ pub fn JournalType(comptime Replica: type, comptime Storage: type) type {
             var dirty_iterator = journal.dirty.bits.iterator(.{ .kind = .set });
             while (dirty_iterator.next()) |dirty_slot| {
                 if (journal.faulty.bit(Slot{ .index = dirty_slot })) continue;
-                assert(journal.prepare_inhabited[dirty_slot]);
-                assert(journal.prepare_checksums[dirty_slot] ==
-                    journal.headers[dirty_slot].checksum);
-                assert(journal.prepare_checksums[dirty_slot] ==
-                    journal.headers_redundant[dirty_slot].checksum);
+                if (journal.prepare_inhabited[dirty_slot]) {
```

Comment:
This was a preexisting bug, discovered by the VOPR when the crash injection was turned up.